monai/visualize/occlusion_sensitivity.py

```diff
@@ -142,139 +71,184 @@ class OcclusionSensitivity:
         - :py:class:`monai.visualize.occlusion_sensitivity.OcclusionSensitivity.`
     """
 
+    @deprecated_arg(
+        name="pad_val",
+        since="1.0",
+        removed="1.1",
```

Comment:
we agreed to remove in the x+2 version, so I think this should be `1.2`: https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#backwards-compatibility
```suggestion
        removed="1.2",
```